src/components/description-data/index.js

```diff
@@ -0,0 +1,61 @@
+import { html, css, LitElement, unsafeCSS } from 'lit';
+import '../grid';
+import styles from './styles.scss';
+import './components/description-data-item';
+
+export class VlDescriptionData extends LitElement {
+  static get styles() {
+    return [
+      css`
+        ${unsafeCSS(styles)}
+      `,
+    ];
+  }
+
+  static get properties() {
+    return {
+      size: { type: String, attribute: 'data-vl-items-size', reflect: true },
+      maxSize: { type: String, attribute: 'data-vl-items-max-size', reflect: true },
+      mediumSize: { type: String, attribute: 'data-vl-items-medium-size', reflect: true },
+      mediumMaxSize: { type: String, attribute: 'data-vl-items-medium-max-size', reflect: true },
+      smallSize: { type: String, attribute: 'data-vl-items-small-size', reflect: true },
+      smallMaxSize: { type: String, attribute: 'data-vl-items-small-max-size', reflect: true },
+      extraSmallSize: { type: String, attribute: 'data-vl-items-extra-small-size', reflect: true },
+      extraSmallMaxSize: { type: String, attribute: 'data-vl-items-extra-small-max-size', reflect: true },
+    };
+  }
+
+  firstUpdated() {
+    const observer = new MutationObserver(() => {
+      this.requestUpdate();
+    });
+
+    observer.observe(this, { subtree: true, childList: true });
+  }
+
+  render() {
+    return html`<div class="vl-description-data">
+      <div is="vl-grid">
+        ${[...this.children].map((child, index) => {
+          const name = `item-${index}`;
+          child.setAttribute('slot', name);
+          return html`<div
+            is="vl-column"
+            data-vl-size=${this.size ? this.size : 12 / this.children.length}
```

Comment:
Ik ben niet zo'n fan van die `data-vl-size=${this.size ? this.size : 12 / this.children.length}`

Ik zou altijd uw gewoon `size` prop reflecteren op dat `data-vl-size` attribuut. Defaults van uw properties kunnen gezet worden in uw constructor. Bv; 
``` 
 constructor() {
    super();
    this.name = 'Somebody';
  }
  
  Voorbeeld => https://lit.dev/